flask_config.py

```diff
@@ -5,4 +5,4 @@ class Config:
     """Base configuration variables."""
     SECRET_KEY = os.environ.get('SECRET_KEY')
     if not SECRET_KEY:
-        raise ValueError("No SECRET_KEY set for Flask application. Did you forget to run setup.sh?")
+        SECRET_KEY = "SOME_KEY"
```

Comment:
If you aren't using cookies (i.e. flask's `session`) then you can remove this rather than having an insecure default value.

Looks like you still have session_items.py but you aren't using it. Delete that file, any mention of it and any mention of SECRET_KEY.

Reply:
done, sorry I missed it